### ISSUE TITLE
Error if npm install was run with python3 as global python

### ISSUE THREAD

**Issue by treyhunner** [ember-cli/ember-cli-deploy#90]
This is really an `execSync` error (issue mgutz/execSync#27), but it affects `ember-cli-deploy`.  Maybe an alternative package could be used to resolve this issue or the issue could be resolved upstream.

```
Cannot find module './build/Release/shell'
Error: Cannot find module './build/Release/shell'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/myrepo/ember/node_modules/ember-cli-deploy/node_modules/execSync/index.js:30:11)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

I don't know the policy for listing issues with upstream packages.  Feel free to close this if it's not appropriate as an ember-cli-deploy issue.

**Comment by joebartels** [comment]
ran into almost the same issue not long ago with another project. Switching to this module, which claims no external dependencies, resolved it.  https://github.com/gvarsanyi/sync-exec

**Comment by joebartels** [comment]
@treyhunner if you have time to test repro'ing that issue with this branch, that'd be awesome. https://github.com/joebartels/ember-cli-deploy/tree/swap_sync_exec

**Comment by lukemelia** [comment]
Thanks for reporting @treyhunner. We hope to get #93 merged soon. If you can confirm it solves your issue, that would be great.

**Comment by treyhunner** [comment]
@lukemelia that branch fixed it for me!

I set the `ember-cli-deploy` version to `git+https://github.com/joebartels/ember-cli-deploy.git#swap_sync_exec`, did an `npm install`, reran `ember deploy -e production` and it works!

**Comment by lukemelia** [comment]
@treyhunner great, we'll get it merged in soon and then get 0.4.1 out.
